wpilib/wpilib/interruptablesensorbase.py

```diff
@@ -73,19 +79,17 @@ def allocateInterrupts(self, watcher):
         """
         if self.interrupt is not None:
             raise ValueError("The interrupt has already been allocated")
-
         self.isSynchronousInterrupt = watcher
         self._interrupt = hal.initializeInterrupts(watcher)
-        self._interrupt_finalizer = weakref.finalize(self, hal.cleanInterrupts,
-                                                     self._interrupt)
+        self._interrupt_finalizer = weakref.finalize(self, hal.cleanInterrupts, self._interrupt)
 
     def cancelInterrupts(self):
         """Cancel interrupts on this device. This deallocates all the
         chipobject structures and disables any interrupts.
         """
         if self.interrupt is None:
             raise ValueError("The interrupt is not allocated.")
-        self._interrupt_finalizer()
+        hal.cleanInterrupts(self.interrupt)
```

Comment:
I wonder what the implications of this change are.

Reply:
i think i was supposed to keep that

Reply:
Yes, you should have kept it, otherwise it'll potentially be called twice.

Reply:
... so put it back?

Reply:
Oh. I'm seeing this comment now